packages/core-utils/src/itinerary.js

```diff
@@ -616,3 +418,13 @@ export function getTimeZoneOffset(itinerary) {
     (new Date().getTimezoneOffset() + dstOffset) * 60000
   );
 }
+
+export function calculateTncFares(itinerary) {
```

Comment:
Well we can't use `calculateFares` anymore since it is deprecated, but this method ensures we can use the deprecated method to get the TNC stuff without returning "dangerous" English strings